src/Uno.UI/Microsoft/UI/Xaml/Controls/Expander/Expander.cs

```diff
@@ -66,10 +70,30 @@ protected override void OnApplyTemplate()
 				}
 			}
 
+			//if (this.GetTemplateChild<Border>(c_expanderContentClip) is Border expanderContentClip)
+			//{
+			//	var visual = ElementCompositionPreview.GetElementVisual(expanderContentClip);
+			//	visual.Clip(visual.Compositor().CreateInsetClip());
+			//}
+
+			if (this.GetTemplateChild<Border>(c_expanderContent) is Border expanderContent)
+			{
+				expanderContent.SizeChanged += OnContentSizeChanged;
```

Comment:
In my PR I added a SerialDisposable to handle removing the event handler here. This is important as I understand things for iOS.